alibi_detect/cd/mmd.py

```diff
@@ -82,27 +89,35 @@ def __init__(
         backend = backend.lower()
         BackendValidator(
             backend_options={'tensorflow': ['tensorflow'],
-                             'pytorch': ['pytorch']},
+                             'pytorch': ['pytorch'],
+                             'keops': ['keops', 'pytorch']},
             construct_name=self.__class__.__name__
         ).verify_backend(backend)
 
         kwargs = locals()
         args = [kwargs['x_ref']]
         pop_kwargs = ['self', 'x_ref', 'backend', '__class__']
+        if backend == 'tensorflow' and has_tensorflow:
+            pop_kwargs += ['device', 'batch_size_permutations']
+            detector = MMDDriftTF
+        elif backend == 'pytorch' and has_pytorch:
+            pop_kwargs += ['batch_size_permutations']
+            detector = MMDDriftTorch
+        else:
+            pop_kwargs += ['configure_kernel_from_x_ref']
```

Comment:
The docstring for this parameter probably needs updating now that it's not used for the KeOps backend (including also a brief explanation why it's not relevant for KeOps).

Reply:
Seems like the explanation on the method page is a lot more comprehensive, we should keep them in sync as much as possible.